samplekotlin/src/main/res/layout/activity_channel_3.xml

```diff
@@ -16,8 +16,9 @@
         android:id="@+id/typingHeaderView"
         android:layout_width="0dp"
         android:layout_height="31dp"
-        android:background="#CCCCCC"
+        android:background="@color/light_gray"
```

Comment:
This small improvement needs too many file modifications, it's not worth complicating the tutorial with 3 file edits to get some colours right. At most we should find one better looking colour for both dark and light mode, and if there isn't one, then we can leave the original.

Reply:
👍